app/src/main/java/com/github/polybooks/FillSaleActivity.kt

```diff
@@ -51,9 +52,33 @@ class FillSaleActivity : AppCompatActivity() {
 
          */
 
-
+        setupNavbar(findViewById(R.id.bottom_navigation))
     }
 
+    private fun setupNavbar(navBar: BottomNavigationView){
+        navBar.selectedItemId = R.id.default_selected
+        navBar.setOnNavigationItemSelectedListener { item ->
+            when(item.itemId){
+                R.id.home -> {
+                    startActivity(Intent(this, MainActivity::class.java))
+                    true
+                }
+                R.id.books ->{
+                    startActivity(Intent(this, FilteringBooksActivity::class.java))
+                    true
+                }
+                R.id.sales ->{
+                    startActivity(Intent(this, FilteringSalesActivity::class.java))
+                    true
+                }
+                R.id.user_profile ->{
+                    // TODO: user sales
+                    false
+                }
+                else -> true
+            }
+        }
+    }
```

Comment:
This function is repeated multiple times in the different files. Can we maybe save it somewhere for later use? Or better, is there a way to say "this navbar is valid for the whole app" with your library? I can't imagine – even knowing how dumb android is – this not being possible